### DISCUSSION TITLE
How would I remove the title from the Navbar?

### DISCUSSION THREAD

**Discussion by FxllenCode** [upptime/upptime#223] [Q&A] [answered]
So I've just started adding branding to my site, and I've been stumped about changing the title from the Navbar. Currently the Navbar looks like this:

![hTEOQIihT5](https://user-images.githubusercontent.com/40151425/105434206-17f0be80-5c29-11eb-8018-127cf7e7be2c.png)

The problem is that extra "Status+", I do not want that to be there as I want to have our full logo, but I can't find how this is done. I've looked under the i18n's but didn't see anything. Does anyone know where I could change this setting? Thanks!

**Comment by AnandChowdhary** [comment]
This title comes from the `name` property here: https://upptime.js.org/docs/configuration#status-website. Maybe you can try an empty string or a space for now? I'll add a configuration option to hide the title, because it's nice to have the title in other places like the `<title>` tag.

**Reply by FxllenCode** [reply]
Yes, that would be great as it makes the title nothing. Thanks!

**Comment by AnandChowdhary** [comment, marked as answer]
You can now hide the title from navbar the using this configuration option:

```yaml
status-website:
  hideNavLogo: false # Hide the logo
  hideNavTitle: true # Hide the title
```
